avocado_vt/plugins/vt.py

```diff
@@ -1180,24 +1185,23 @@ def configure(self, parser):
                                                   "If --vt-config is "
                                                   "provided, this will be "
                                                   "ignored. Default: ''"))
-        qemu_bin_path_current = settings.get_value('vt.qemu', 'qemu_bin',
-                                                   default=qemu_bin_path)
+        qemu_bin = settings.get_value('vt.qemu', 'qemu_bin',
```

Comment:
I'm not sure what you're proposing, can you please elaborate it a bit? The problem in the original version is that it sets error string as default value, which makes avocado fail even though it can perfectly run with None.